bcda-site-static/index.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: home
+title:  "Beneficiary Claims Data API"
+date:   2017-10-30 09:21:12 -0500
+description: Shared Savings Program ACOs can use this FHIR-based API to retrieve bulk claims data related to their assigned beneficiaries. Under construction; feedback invited.
```

Comment:
Just double-checking we want the construction message here

Reply:
Elissa confirmed these are the approved words.